app/controllers/menu_controller.rb

```diff
@@ -0,0 +1,8 @@
+class MenuController < ApplicationController
+    def index
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.